src/components/Image/index.native.js

```diff
@@ -42,4 +55,5 @@ const ImageWithOnyx = withOnyx({
     },
 })(Image);
 ImageWithOnyx.resizeMode = RESIZE_MODES;
+ImageWithOnyx.resolveDimensions = resolveDimensions;
```

Comment:
Hey @chrispader, @terrysahaidak,

While refactoring the Image component, I came across the `resolveDimensions` function you added. I have a few questions:

1. What is the intended purpose of `resolveDimensions`? I couldn't find any calls to it in our current codebase (both currently and historically).
2. The function seems to be present only in the native component. Shouldn't it be available in both the native and web? This is important since the code might not be aware of which Image component (native or web) is currently in use.
3. Upon checking the external sources for both `Image` and `FastImage`, I couldn't find any mention or implementation of `resolveDimensions`.
4. Is this function still in progress, or is it perhaps a legacy piece of code that can be safely removed?

Thank you for your insights.

Reply:
No, you're right, this is in fact a leftover from previous changes. This can safely be removed!

Should i create a PR for this or are you including this in some other PR? :)

Reply:
Thanks for the reply, no need to do anything - I'm removing the code as part of the refactor here
- #13036